model/nextcloud/nextcloud.go

```diff
@@ -99,48 +100,54 @@ func New(inst *instance.Instance, accountID string) (*NextCloud, error) {
 		Host:     u.Host,
 		Username: username,
 		Password: password,
+		BasePath: "/remote.php/dav",
 		Logger:   logger,
 	}
 	nc := &NextCloud{
 		inst:      inst,
 		accountID: accountID,
 		webdav:    webdav,
 	}
-	if err := nc.fillBasePath(&doc); err != nil {
+	if err := nc.fillUserID(&doc); err != nil {
 		return nil, err
 	}
 	return nc, nil
 }
 
 func (nc *NextCloud) Download(path string) (*webdav.Download, error) {
-	return nc.webdav.Get(path)
+	return nc.webdav.Get("/files/" + nc.userID + "/" + path)
 }
 
 func (nc *NextCloud) Upload(path, mime string, contentLength int64, body io.Reader) error {
 	headers := map[string]string{
 		echo.HeaderContentType: mime,
 	}
+	path = "/files/" + nc.userID + "/" + path
```

Comment:
Looks like it could be the right time to add a `filePath(path string)` and a `trashedPath(path string)` methods?!